docs/modules/ROOT/pages/how-to/connect-gitlab.adoc

```diff
@@ -0,0 +1,121 @@
+. Fill the form with the following information:
+** Kubernetes cluster name: "APPUiO Cloud [ZONE NAME]."
+** API URL: Use the xref:references/zones.adoc[URL corresponding to your chosen zone].
+** CA Certificate: leave empty.
+** Service Token: paste the value copied in the `oc get secret` command above.
+** Check the "RBAC-enabled cluster" option.
+** Uncheck "GitLab-managed cluster" and "Namespace per environment."
+** Leave the "Project namespace prefix (optional, unique)" empty.
+. Click the btn[Add Kubernetes cluster]
+
+image::gitlab-kubernetes.png[]
+
+IMPORTANT: If the token is invalid, the message _"There was a problem authenticating with your cluster. Please ensure your CA Certificate and Token are valid"_ will appear. Check that the pasted token value is correct.
+
+== Connecting APPUiO Cloud to the GitLab Container Registry
+
+To configure your GitLab container registry in your APPUiO Cloud project, follow these steps:
+
+. Create a deploy token in GitLab.
+. Create a secret in APPUiO Cloud.
+. Specify the secret in your deployment.
+
+=== Create a Deploy Token in GitLab
+
+Follow these steps to create a new deploy token in your GitLab project:
+
+. In your GitLab project, select and expand the menu:Settings[Repository > Deploy Tokens] item, and scroll to the bottom to select the btn:[Connect existing cluster] button.
+. Provide a name, an (optional) expiration date, and a an (optional) usename.
+. Check the `read_registry` checkbox.
+. Click the btn:[Create deploy token] button.
+. Copy the token immediately; it won't be shown ever again.
+
+image::gitlab-registry.png[]
+
+NOTE: If no name is provided, GitLab provides a default username of the form `gitlab+deploy-token-{n}`.
+
+=== Create a Secret
+
+The `oc` command below creates a secret with the required values. Replace the `--docker-username` and `--docker-password` values with those used in the previous step. Also replace the `--docker-server` parameter with the URL of your own GitLab instance if you have one.
+
+[source,shell]
+----
+$ oc create secret docker-registry gitlab-pull-secret \
+	--docker-server=https://registry.gitlab.com \
+	--docker-username=your-token-username \   # <1>
+    --docker-password=xXXxXxxXXXxXXXxXXXxX    # <2>
+----
+<1> The username is the same one entered in the GitLab token form shown above in step 2, or else the default name generated by GitLab.
+<2> The value of the token, copied in step 5 above.
+
+=== Specify the Secret in your Deployment
+
+Use the YAML below to specify your new secret.
+
+[source,yaml]
+----
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: deployment-name
+spec:
+  template:
+    metadata:
+      labels:
+        app: app-name
+    spec:
+      imagePullSecrets:
+        - name: gitlab-pull-secret # <1>
+      containers:
+        - name: xxxx
+          image: xxxxxxxxxx
+          # …
+----
+
+<1> Name of the secret created in the previous step.
```

Comment:
Maybe only use the minimal path to spec?
Besides, we should probably not encourage the use of `DeploymentConfig` anymore?
```suggestion
apiVersion: apps/v1
kind: Deployment
metadata:
  name: deployment-name
spec:
  template:
    spec:
      imagePullSecrets:
        - name: gitlab-pull-secret # <1>
      ...
----
<1> Name of the secret created in the previous step.
```